charts/performancetesting/templates/cron.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ .Chart.Name }}-cron
+  namespace: {{ .Values.namespace }}
+  labels:
+    app.kubernetes.io/name: uosjarchart
+spec:
+  schedule: "{{ .Values.schedule }}"
+  concurrencyPolicy: Forbid
+  suspend: false
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: {{ .Chart.Name }}-cron
+            image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+            imagePullPolicy: {{ .Values.image.pullPolicy }}
+            requests:
+              memory: {{ .Values.resources.requests.memory }}
+              cpu: {{ .Values.resources.requests.cpu }}
+            limits:
+              memory: {{ .Values.resources.limits.memory }}
+              cpu: {{ .Values.resources.limits.cpu }}
+            ports:
+            - containerPort: {{ .Values.service.ports }}
```

Comment:
I see that this will map to port 8080, as stated in the `values.yml` file.

Does this container need an open port? Maybe I need a reminder of where the result of the test will be placed.

Reply:
Talking with Victor we decided to delete ports, if we need in the future we can add it.